install.sh

```diff
@@ -47,8 +47,11 @@ RPM=$(which yum || :)
 # Detect SystemV / SystemD
 SYS=$([[ -L "/sbin/init" ]] && echo 'SystemD' || echo 'SystemV')
 
+# Detect GLIBC version
+GLIBC_VERSION=$(ldd --version | grep ldd | awk '{print $NF}')
+
 # Detect Debian/Ubuntu derivative
-DEB_ID=$( (grep DISTRIB_CODENAME /etc/upstream-release/lsb-release || grep DISTRIB_CODENAME /etc/lsb-release) 2>/dev/null | cut -d'=' -f2 )
+DEB_ID=$( (grep DISTRIB_CODENAME /etc/upstream-release/lsb-release || grep DISTRIB_CODENAME /etc/lsb-release || grep VERSION_CODENAME /etc/os-release)  2>/dev/null | cut -d'=' -f2 )
```

Comment:
Why not just `lsb_release --short --codename`?

Reply:
Some Ubuntu derivatives would fail since this would return the derivative codename itself. That for sure was the case with Mint for example.